Tutorial.md

```diff
@@ -11,477 +12,247 @@ Gist is a paste. A snippet of text. Not unrelated to [GitHub Gists][].  To set s
-
-## Collection Resource
-With single the Gist Resource in place let's define the collection of our Gist Resources:
-
-    ## Gists Collection [/gists{?since}]
-    Collection of all Gists.
-
-    The Gist Collection resource has the following attribute:
-
-    - total
-
-    In addition it **embeds** *Gist Resources* in the Gist Fox API.
-
-
-    + Model (application/hal+json)
+This example returns a 201 status code, along with HTTP headers and a JSON Body.
 
-        HAL+JSON representation of Gist Collection Resource. The Gist resources in collections are embedded. Note the embedded Gists resource are incomplete representations of the Gist in question. Use the respective Gist link to retrieve its full representation.
+```markdown
++ Response 201 (application/json)
 
-        + Headers
+    + Headers
 
-                Link: <http:/api.gistfox.com/gists>;rel="self"
+            Location: /questions/1
 
-        + Body
+    + Body
 
                 {
-                    "_links": {
-                        "self": { "href": "/gists" }
-                    },
-                    "_embedded": {
-                        "gists": [
-                            {
-                                "_links" : {
-                                    "self": { "href": "/gists/42" }
-                                },
-                                "id": "42",
-                                "created_at": "2014-04-14T02:15:15Z",
-                                "description": "Description of Gist"
-                            }
-                        ]
-                    },
-                    "total": 1
+                    "question": "Favourite programming language?",
+                    "published_at": "2014-11-11T08:40:51.620Z",
+                    "url": "/questions/1",
+                    "choices": [
+                        {
+                            "choice": "Swift",
+                            "url": "/questions/1/choices/1",
+                            "votes": 0
+                        }, {
+                            "choice": "Python",
+                            "url": "/questions/1/choices/2",
+                            "votes": 0
+                        }, {
+                            "choice": "Objective-C",
+                            "url": "/questions/1/choices/3",
+                            "votes": 0
+                        }, {
+                            "choice": "Ruby",
+                            "url": "/questions/1/choices/4",
+                            "votes": 0
+                        }
+                    ]
                 }
+```
 
-    ### List All Gists [GET]
-    + Parameters
-        + since (optional, string) ... Timestamp in ISO 8601 format: `YYYY-MM-DDTHH:MM:SSZ` Only Gists updated at or after this time are returned.
+The next resource is “Question”, it allows you to view a single question directly with an ID of a “Question”.
```

Comment:
"an ID of a “Question”" :arrow_right "its ID" or "using its ID", but frankly I would not mention it (the ID) here 

Something like:

"The next resource is “Question”, which it allows you to view a single question."

Or better, if you do not need to make a claim about this resource actions: 

"The next resource is “Question”, which represents a single question."